src/sidecar/sidecarManager.ts

```diff
@@ -500,13 +501,15 @@ export class SidecarManager {
           false,
         );
         if (notifySidecarExceptions) {
-          vscode.window
-            .showErrorMessage(`Sidecar error: ${errorMatch[2]}`, "Open Logs")
-            .then((action) => {
-              if (action === "Open Logs") {
-                vscode.commands.executeCommand("confluent.showSidecarOutputChannel");
-              }
-            });
+          showErrorNotificationWithButtons(`[Debugging] Sidecar error: ${errorMatch[2]}`, {
+            "Open Sidecar Logs": () =>
+              vscode.commands.executeCommand("confluent.showSidecarOutputChannel"),
+            "Open Settings": () =>
+              vscode.commands.executeCommand(
+                "workbench.action.openSettings",
+                "@id:confluent.debugging.showSidecarExceptions",
+              ),
+          });
```

Comment:
Example of the "custom buttons + callbacks" implementation for `showErrorNotificationWithButtons()`